fastlane/Fastfile

```diff
@@ -105,33 +105,14 @@ platform :android do
 
     firebase_app_distribution(
       app: "1:867301491091:android:50b626dba42a361651e866",
-      android_artifact_type: "APK",
-      android_artifact_path: "app/build/outputs/apk/release/com.bitwarden.authenticator-release.apk",
+      android_artifact_type: "AAB",
+      android_artifact_path: "app/build/outputs/bundle/release/com.bitwarden.authenticator-release.aab",
       service_credentials_file: options[:serviceCredentialsFile],
       groups: "internal-prod-group",
       release_notes: release_notes,
     )
   end
 
-    desc "Publish release AAB to Firebase"
```

Comment:
The diff looks kinda funny here because the indentation was wrong and I corrected it. I can revert this so it looks more normal and fix the indention in a different PR if you'd like.